pagopa/io-app
# fix: [PE-725] CGN merchant categories list details transition header

## Short description
This pull request includes several changes to the `CgnMerchantCategoriesListScreen.tsx` and `categories.ts` files to improve the handling of the first render state and loading indicators.
## List of changes proposed in this pull request
- Update CGN categories store loading state to `noneLoading`
- Ensure the `RefreshControl` component does not appear on the first render.

## How to test
- Enter into CGN detail screen
- Press on _Scopri le opportunità_
- Check the list transition
